concepts/sbtc/peg-wallet-utxo.md

```diff
@@ -0,0 +1,13 @@
+# Peg Wallet UTXO
+
+The Peg Wallet UTXO is a fundamental element of the sBTC system, serving as the Bitcoin backing for all sBTC tokens in circulation. This system employs a Single UTXO Model, where the sBTC peg wallet is consistently represented as a single Unspent Transaction Output (UTXO) on the Bitcoin blockchain. This design choice offers simplicity and improved efficiency in managing the peg wallet.
+
+UTXO management falls under the responsibility of the Signer coordinator. This entity consolidates all deposit and withdrawal requests, creating optimized batches that can be processed within a single UTXO. The new UTXO is created by spending the amount from the previous UTXO, adding confirmed deposits, and subtracting confirmed withdrawals.
```

Comment:
I don't like the use of "This entity" here. can it be rephrased like "The set of sBTC signers will consolidate..."?

Reply:
@AshtonStephens who actually does consolidate the UTXOs? Is this the responsibility of the signer set or a single signer coordinator?

Reply:
i think that's my concern precisely. "this entity" makes it sounds like it's a single entity that does this, vs the full set of signers

Reply:
It's the set of signers.

The Signer coordinator makes the UTXO and then the others need to approve it by signing it for it to be valid. So a single signer sort of "makes" it, but the rest partake in the making by signing.